src/v2/recommender.py

```diff
@@ -465,8 +465,8 @@ def execute(self, arguments=None, persist=True, check_license=False):
         ended_at = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")
         audit = {'started_at': started_at, 'ended_at': ended_at, 'version': 'v2'}
 
-        recommendation = StackRecommendationResult(**recommendation,
-                                                   **request.dict()).dict()
+        recommendation = json.loads(StackRecommendationResult(**recommendation,
+                                                              **request.dict()).json())
```

Comment:
Why the dictionary is converted to json and again converted to dictionary?

Reply:
This was required as Model.dict() was not supporting conversion of UUID into string. Only easy way I could find was to get JSON and then create dict out of JSON. Let me know if you have any suggestions to try our here.
If we don't convert UUID, then final response builder which jsonifies the dict object throws error.

Reply:
Use pydantic native UUID4 type.

`from pydantic import UUID4`

Reply:
Pydantic UUID4 also gives conversion error. Treating uuid as string as mutually agreed.